pkg/server.go

```diff
@@ -102,30 +143,63 @@ func Start(cfg *Config) {
 	}
 
 	if tlsEnabled {
-		log.Infof("listening on https://:%d", cfg.Port)
+		log.Infof("listening on https. port: %d", cfg.Port)
+
+		if acmMode {
+			startProxy(cfg, k8sclient, tlsConfig, timeout, proxy.AlertManagerKind, proxy.AlertmanagerPort)
+			startProxy(cfg, k8sclient, tlsConfig, timeout, proxy.ThanosQuerierKind, proxy.ThanosQuerierPort)
+		}
+
 		logrus.SetLevel(logrusLevel)
 		panic(httpServer.ListenAndServeTLS(cfg.CertFile, cfg.PrivateKeyFile))
 	} else {
-		log.Infof("listening on http://:%d", cfg.Port)
+		log.Infof("listening on http. port: %d", cfg.Port)
 		logrus.SetLevel(logrusLevel)
 		panic(httpServer.ListenAndServe())
 	}
 }
 
-func setupRoutes(cfg *Config) (*mux.Router, *PluginConfig) {
+func setupRoutes(cfg *Config, acmMode bool) (*mux.Router, *PluginConfig) {
```

Comment:
does the cfg hold the features already, we will have more features like incidents

Reply:
It had made the logic a bit easier, but I agree that adding it doesn't scale too well as we add more features. I'll remove it